package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "tabletop.js",
-  "version": "1.0.0",
+  "name": "tabletop-boardgames",
+  "version": "1.0.2",
```

Comment:
I was thinking that might be the case. 0.1.0 was what I was thinking.. Just to indicate this is not a release version